packages/console/sst.config.ts

```diff
@@ -51,7 +51,7 @@ export default $config({
     const api = new sst.aws.Function(`${PROJECT_NAME}-api`, {
       vpc,
       handler: "bootstrap",
-      bundle: "target/lambda/opraas_server",
+      bundle: "target/lambda/opraas_console_server",
```

Comment:
are you sure about this? You can try building the server package and inspecting target... 

Also I think you need to update `/Cargo.toml` with the new server path. And the `justfile`. This will derive from those updates